src/runtime_src/core/include/experimental/xrt_ip.h

```diff
@@ -44,6 +46,18 @@ class ip_impl;
 class ip : public detail::pimpl<ip_impl>
 {
 public:
+  static constexpr size_t isr_size = 32;
+  /*!
+   * @interrupt_vector
+   *
+   * @brief
+   * xrt::ip::interrupt_vector represents an IP interrupt vector object 
+   * with value from ISR register.
+   * This object is passed to callback function as argument
+   * This has 32 bits
```

Comment:
The 32 bits corresponds to what?  I know of course, but does the reader?

Reply:
updated